### PR TITLE
Calculator Desgin

### PR DESCRIPTION
**Pull request by arabiu033**
# Changes Highlighted :bookmark_tabs: 

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- In Calculator.js, create a React component
- Add some js files to support the Calculator.js
- Fix Eslint errors and linters
- Make project match the requirement.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calc from './components/Calculator';
 
 function App() {
   return (
```
Comment (AmaduKamara) [inline src/App.js]: - [x] Kindly make your App.js component a class-based component. It's a requirement that these components be class-based components. Especially the Calculator and App components should be class-based for now 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @arabiu033,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly
✅ Linter checks passed 
✅ Good descriptive PR 👍 
✅ Your app is nicely designed 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 1 above)
Comment (arabiu033) [reply src/App.js]: Ok.

**Review by ShahierNashaat — APPROVED** [review]
Hi @arabiu033, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**